gradle/wrapper/gradle-wrapper.properties

```diff
@@ -0,0 +1,5 @@
+distributionBase=GRADLE_USER_HOME
+distributionPath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.6.1-all.zip
+zipStoreBase=GRADLE_USER_HOME
+zipStorePath=wrapper/dists
```

Comment:
I know it's common practice among some people to check in gradle wrapper and maven wrapper stuff, but I have some objections to this practice:

1. The project should build fine without these, and consumers of the software should not be required to use them if they do not wish to,
2. The checked-in code is static, and does not reflect upstream, current wrapper code, so inlining it into the project ensures it's going to be out of date at some point, and most importantly,
3. The checked in jar is binary, and therefore fails to meet either the OSD's definition of "open source" or the FSF's definition of "free" software. I'm strongly opposed to including non-open source and/or non-free software in a repository that purports to be open source software.

Reply:
that was unintended, I think it is because I added them before adding the git ignore